graphql_compiler/tests/snapshot_tests/test_orientdb_match_query.py

```diff
@@ -796,5 +823,4 @@ def test_filter_within_fold_scope(self):
 
         self.assertMatchSnapshot(rows)
 
-
```

Comment:
nit: don't format lines unrelated to diff.

Reply:
done!